rfcs/0016-permissionless-assets-for-ibc-and-cw.md

```diff
@@ -0,0 +1,217 @@
+# Overview
+
+New assets, not registered before, can be transferred via Bridges to Dotsama and created by permissionless transactions, including  CosmWasm contracts, without any prior registration or permissioned governance.
+
+Also all permissionless asset still can be governed by Root, they also have admin accounts which are set by by permissionless transactions on creation for source assets (originated from our chains) or set by Root for bridged assets for permissionless updates of some properties.
+
+To create new permissionless asset `CreationFee` must be paid.
+
+## Target semantic data structure
+
+```typescript
+// none of these ever set by permissionless transactions/users
+interface AssetId {
+    // 4 byte number, set once per each consensus connection
+    // asset from specific network transferred via XCM or IBC will have same network_id,
+    // but different protocol_id
+    // picasso id = 0
+    network_id : number
+    // 8 byte number, hardcoded into runtime codebase
+    // examples, are DEX, CW, IBC, XCM, XCVM
+    // use pallet identifier where it is fit
+    protocol_id : number
+    // 4 byte number, incremented predictably by assets-registry or by cross chain governance per protocol_id
+    nonce: number
+}
+
+
+interface GovernanceSufficiency {
+    /// when set allows to pay ED in this asset
+    /// replaced by DEX oracle and some storage multiplier later
+    existential_deposit: number?
+    /// ratio of PICA token to this assets
```

Comment:
Are we doing ED in PICA now?

Reply:
Will clarify that.  
Current state - any sufficient assets pays for self.
Easy to do - any non sufficient asset ED is payed in PICA
Harder - any non sufficient asset pays ED in any sufficinet - I guess this may be target, but anyway needs things from previous thep to be here, and harded to do.
And permissionless asset is non sufficient on creation.

Reply:
making Deposits in PICA simpler and  is faster time to market